src/Controls/src/Core/BindingExpression.cs

```diff
@@ -463,6 +464,13 @@ internal static bool TryConvert(ref object value, BindableProperty targetPropert
 					return false;
 				}
 
+				// If the input string is null, empty, or consists only of whitespace, 
+				// and the target type is a numeric type, set the value to 0 to ensure proper numeric conversion
+				if (string.IsNullOrWhiteSpace(stringValue) && NumericTypes.Contains(convertTo))
```

Comment:
This is maybe better than before where `""` now equals `0`, but how would you type `-3`?

When you clear the text box, it adds a 0. So, I am thinking it may be impossible?

What happens of WPF and/or WinUI?

Reply:
Hi, @mattleibow typing a negative number is possible :)


https://github.com/dotnet/maui/assets/42434498/e5817fb4-75f1-49b1-93b0-a0fd5ec77413

Reply:
That is not really a normal way of typing :)